packages/react-lowcode/src/codegen/generation/entity/index.ts

```diff
@@ -1,4 +1,6 @@
 export interface Entity {
+  getListTypeQueryName(): string | undefined,
+  getDetailTypeQueryName(): string | undefined,
```

Comment:
this is not a proper place - let's keep this interface simple
please add introspection to template resolver constructor and col some function from `graphql-codegen` instead